src/protocols/eth.rs

```diff
@@ -234,19 +234,81 @@ pub fn parse_block_bodies(payload: Vec<u8>) -> Vec<Vec<Transaction>> {
 
         let transactions = block_body.at(0).unwrap();
         let count_tx = transactions.item_count().unwrap();
+        let ommers = block_body.at(1).unwrap();
+        let count_om = ommers.item_count().unwrap();
 
         trace!("Transactions count : {}", count_tx);
 
         let mut result_transactions: Vec<Transaction> = vec![];
+        let mut result_ommers: Vec<Block> = vec![];
 
         for j in 0..count_tx {
             let transaction = transactions.at(j).unwrap();
             let t = parse_transaction(transaction.as_raw().to_vec());
             result_transactions.push(t);
         }
 
-        result.push(result_transactions);
+        for k in 0..count_om {
+            let ommer = ommers.at(k).unwrap();
+            let om = parse_block_header(ommer.as_raw().to_vec());
+            result_ommers.push(om);
+        }
+
+        result.push((result_transactions, result_ommers));
     }
 
     return result;
 }
+
+pub fn parse_block_header(payload: Vec<u8>) -> Block {
```

Comment:
We should use this function to parse Block Header in the `parse_block_headers` starting at line 143 to avoid duplicate code.